evaluate/eval.py

```diff
@@ -77,12 +77,18 @@ def evaluate_repairs(self):
 
     def eval_report(self):
         tic = time.clock()
-        prec, rec, rep_recall, f1, rep_f1 = self.evaluate_repairs()
-        report = "Precision = %.2f, Recall = %.2f, Repairing Recall = %.2f, F1 = %.2f, Repairing F1 = %.2f, Detected Errors = %d, Total Errors = %d, Correct Repairs = %d, Total Repairs = %d, Total Repairs (Grdth present) = %d" % (
-                  prec, rec, rep_recall, f1, rep_f1, self.detected_errors, self.total_errors, self.correct_repairs, self.total_repairs, self.total_repairs_grdt)
+        try:
+            prec, rec, rep_recall, f1, rep_f1 = self.evaluate_repairs()
+            report = "Precision = %.2f, Recall = %.2f, Repairing Recall = %.2f, F1 = %.2f, Repairing F1 = %.2f, Detected Errors = %d, Total Errors = %d, Correct Repairs = %d, Total Repairs = %d, Total Repairs (Grdth present) = %d" % (
+                      prec, rec, rep_recall, f1, rep_f1, self.detected_errors, self.total_errors, self.correct_repairs, self.total_repairs, self.total_repairs_grdt)
+            report_list = [prec, rec, rep_recall, f1, rep_f1, self.detected_errors, self.total_errors,
+                           self.correct_repairs, self.total_repairs, self.total_repairs_grdt]
+        except Exception as e:
+            logging.error("ERROR generating evaluation report %s" % e)
+            raise Exception("ERROR generating evaluation report")
```

Comment:
You should re-raise this instead of raising another exception, otherwise the stack trace will be destroyed, i.e.
```
raise
```
with no arguments (since you've already logged the error message).